modules/explorer-core/src/main/scala/org/ergoplatform/explorer/db/queries/DexOrdersQuerySet.scala

```diff
@@ -0,0 +1,73 @@
+package org.ergoplatform.explorer.db.queries
+
+import doobie.Fragments
+import doobie.util.query.Query0
+import doobie.implicits._
+import doobie.refined.implicits._
+import org.ergoplatform.explorer._
+import org.ergoplatform.explorer.db.models.{Asset, Input, Output}
+import org.ergoplatform.explorer.db.models.aggregates.ExtendedOutput
+
+/** A set of queries for doobie implementation of [DexOrdersRepo].
+  */
+object DexOrdersQuerySet {
```

Comment:
Queries are grouped by entity type they work with. These queries deal with `ExtendedOutput` which is an aggregate derived mainly from `Output`, so the current architecture implies that these queries should be in `OutputQuerySet`.

Reply:
Done.